### PR TITLE
Adding pattern to cover all patch versions

### PR DESCRIPTION
**Pull request by mariantiris**
Adding pattern to cover all API versions: v3.1(.[1-9])?

What will match examples:

v3.1
v3.1.1 ... v3.1.9

### CODE REVIEW

File: config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name" : "10 - Open Banking Account Access Consent",
   "auditService": "AuditService-OB-Route",
   "baseURI" : "https://&{iam.fqdn}",
-  "condition" : "${matches(request.uri.path, '^/rs/open-banking/v3.1/aisp/account-access-consents')}",
+  "condition" : "${matches(request.uri.path, '^/rs/open-banking/v3.1(.[1-9])?/aisp/account-access-consents')}",
```
Comment (BohoCode) [inline config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json:6]: would that match v3.1.0? Or v3.1.10?

Comment (mariantiris) [reply config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json:6]: At the moment no. It matches only 3.1 and every patch version in between 3.1.1 and 3.1.9 including them.

Comment (jorgesanchezperez) [reply config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json:6]: What thoughts do you have about the below patter?
`v(\\\\d+.)?(\\\\d+.)?(\\\\*|\\\\d+)`
This pattern will support the bellow strings (where X Y and Z are digits):
`'/rs/open-banking/vX/aisp/accounts'`
`'/rs/open-banking/vX.Y/aisp/accounts'`
`'/rs/open-banking/vX.Y.Z/aisp/accounts'`
`'/rs/open-banking/V/aisp/accounts'`  not supported 404
`'/rs/open-banking/X/aisp/accounts'`  not supported 404
`'/rs/open-banking/v/aisp/accounts'`  not supported 404
`'/rs/open-banking/vX./aisp/accounts'` not supported 404
`'/rs/open-banking/vX.Y./aisp/accounts'` not supported 404
`'/rs/open-banking/vX.Y.Z./aisp/accounts'` not supported 404
Any pattern not supported 404

Comment (jorgesanchezperez) [reply config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json:6]: You can test the pattern `v(\d+.)?(\d+.)?(\*|\d+)` on https://regex101.com with value `/something/v3.1.1500`

Comment (jorgesanchezperez) [reply config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json:6]: Another patter could be `v3.1(\.\d+)?` for IG `(\\\\.\\\\d+)?`

Comment (jorgesanchezperez) [reply config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json:6]: In my opinion I think that the pattern `v3.1(\.\d+)?` is enough, but this depends of customers, we need to wait what is the low version used by our customers currently or prospect customers.
@BohoCode is working to know that.

**Review by BohoCode — APPROVED** [review]

**Review by BohoCode — APPROVED** [review]

**Review by jorgesanchezperez — APPROVED** [review]

**Pull request merged**